pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -267,11 +267,7 @@ func (admitter *VMsAdmitter) applyInstancetypeToVm(vm *v1.VirtualMachine) (*inst
 		}}
 	}
 
-	if instancetypeSpec == nil && preferenceSpec == nil {
-		return nil, nil, nil
-	}
-
-	if topology := instancetype.GetPreferredTopology(preferenceSpec); topology == instancetypev1beta1.PreferSpread {
+	if topology := instancetype.GetPreferredTopology(preferenceSpec); instancetypeSpec != nil && topology == instancetypev1beta1.PreferSpread {
```

Comment:
nit: can we split this into 2 lines. I find it a bit complex with 2 conditions

Reply:
I wanted to, but `make format` did have another opinion. Want me to move it out of the if condition?

Reply:
Ah I see, mh... do you know if at a certain point we want to make it mandatory or are those only suggestions?

Reply:
Anyway, it is just a nit comment, feel free to ignore it

Reply:
I think it is already sort of mandatory. Once you run `make format` it will format the whole code base (not just your changes) and IDE's usually run formatting on every save too.

Reply:
ok, then let's leave it like this